.changeset/unlucky-rivers-grow.md

```diff
@@ -0,0 +1,10 @@
+---
+"@ledgerhq/hw-app-aptos": patch
+"@ledgerhq/types-live": patch
+"@ledgerhq/coin-aptos": patch
+"ledger-live-desktop": patch
+"@ledgerhq/live-common": patch
+"@ledgerhq/coin-framework": patch
```

Comment:
Why is there so many impacted packages shouldn't it just be `ledger-live-desktop` ?